### ISSUE TITLE
Missing salt-doc packages for 2015.8.0 in the new ubuntu repo

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#27456]
Happily upgraded to 2015.8.0 but the salt-doc package is missing from the new repo : 

```
# dpkg -l *salt*
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                              Version               Architecture          Description
+++-=================================-=====================-=====================-========================================================================
ii  salt-api                          2015.8.0+ds-1         all                   Generic, modular network access system
ii  salt-cloud                        2015.8.0+ds-1         all                   public cloud VM management system
ii  salt-common                       2015.8.0+ds-1         all                   shared libraries that salt requires for all packages
ii  salt-doc                          2015.5.2+ds-1trusty1  all                   additional documentation for salt, the distributed remote execution syst
ii  salt-master                       2015.8.0+ds-1         all                   remote manager to administer servers via salt
ii  salt-minion                       2015.8.0+ds-1         all                   client package for salt, the distributed remote execution system
ii  salt-ssh                          2015.8.0+ds-1         all                   remote manager to administer servers via salt
ii  salt-syndic                       2015.8.0+ds-1         all                   master-of-masters for salt, the distributed remote execution system
```

**Comment by jfindlay** [comment]
@arthurlogilab, thanks for the report.

**Comment by dmurphy18** [comment]
@arthurlogilab Some issues occurred with sphinx 1.3 not being available which salt 2015.8.0 required for generating the documentation. The easiest solution, given time constraints, was to disable the generation of the documentation.  I shall take a further look at enabling the documentation on Ubuntu 14.04 and 12.04 platforms.  Note that the Debian packages work with Ubuntu 15.04, since both support systemd and documentation is present there.

**Comment by whiteinge** [comment]
@dmurphy18 Since our self-hosted packages don't need to adhere to purity standards, we can (more) simply take @jacobhammons 's pre-built docs and just package the static files. No need to try to figure out how to build the docs on each and every OS we package for.
